docs/admin/access.rst

```diff
@@ -224,6 +224,9 @@ turned on through the project page. The interface allows you to:
 * Change permissions of the users
 * Revoke access to the users
 
+.. versionadded:: 4.2.1
+* Resend user email invitation, invalidating the any previous sent invitation
```

Comment:
I don't think `versionadded` can be reasonably used inside list, so I'd probably omit that.

Reply:
awesome thanks for the hint